source/manual/lint-ruby-code

```diff
@@ -0,0 +1,31 @@
+---
+owner_slack: "#govuk-developers"
+title: Lint your Ruby code with govuk-lint
+section: Patterns & Style Guides
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-10-03
+review_in: 6 months
+---
+
+The [govuk-lint](https://github.com/alphagov/govuk-lint) gem uses Rubocop to enforce consistency with the GOV.UK style guide.
+
+Rubocop is an [open-source gem](https://github.com/bbatsov/rubocop) that performs static analysis of Ruby code according to rules that can be granularly configured. Each validation, or rule, is called a "cop". Some of those cops come with the ability to auto-correct issues.
```

Comment:
Perhaps:

`[Rubocop](https://github.com/bbatsov/rubocop) is an open-source gem`

instead?

Reply:
👍